drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c

```diff
@@ -152,7 +203,19 @@ int lsm6dsv16x_trigger_set(const struct device *dev,
 				lsm6dsv16x_enable_g_int(dev, LSM6DSV16X_DIS_BIT);
 			}
 		}
-
+		break;
+	case SENSOR_TRIG_DELTA:
+		if (trig->chan == SENSOR_CHAN_ACCEL_XYZ) {
```

Comment:
```suggestion
		if (trig->chan != SENSOR_CHAN_ACCEL_XYZ) {
        return -ENOTSUP;

```
Invert the condition to reduce nested if😊